assets/css/home.css

```diff
@@ -35,14 +35,17 @@ html {
 }
 .content h1 {
     @apply text-2xl md:text-4xl font-bold mb-3;
+		margin-bottom: 1.2em;
```

Comment:
As we're using tailwindcss. Lets change this one to be `@apply mb-1` or mb1.5 depending of what works best.

Reply:
Ah, I heard one of my colleagues talk about tailwind something! maybe this? committed!